backend/app/assets/stylesheets/spree/backend/_bootstrap_custom.scss

```diff
@@ -289,16 +289,16 @@ $input-padding-y:                .375rem !default;
 $input-bg:                       #fff !default;
 $input-bg-disabled:              $gray-lighter !default;
 
-$input-color:                    $gray !default;
-$input-border-color:             #ccc !default;
+$input-color:                    $color-txt-text !default;
```

Comment:
We've deprecated it, which means if a user sets `$color-txt-text`, they should see a warning, but it should continue working. This is the line that allows it to continue working